.github/pull_request_template.md

```diff
@@ -1,25 +1,14 @@
 <!-- if needed please write above the given line -->
+### Thanks for your PR! Please fill out the following list :)
+
 ---
-Release Notes:
+- [ ] I put the ticket in review
+- [ ] Sonarcloud link : \<add the link here>
+- [ ] No documentation update needed
+- [ ] Link to PR with documentation update: \<add the link here>
+- [ ] No Release Notes needed
+- [ ] Release Notes :
```

Comment:
Do we use those release notes while building a release?

Reply:
We sometimes use them for bigger tickets, because the commit messages don't suffice.